packages/main/src/plugin/api/informer-info.ts

```diff
@@ -0,0 +1,26 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import type { Context, Informer, KubernetesObject } from '@kubernetes/client-node';
+
+export type InformerResourcesType = 'INGRESS';
+
+export interface InformerInfo {
+  informer: Informer<KubernetesObject>;
+  context: Context;
+  resourcesType: InformerResourcesType;
+}
```

Comment:
I don't see why InformerResourcesType or InformerInfo would container Kubernetes or Ingress types

IMHO it looks like such generic name can't be used with such specific data added (should be generic and do not contain any Kubernetes or INGRESSES values there)

Reply:
should it be a KubernetesInformer* to be more accurate ? / filename changed as well